train.py

```diff
@@ -0,0 +1,22 @@
+"""Train file for training language models with sweeps
+Inspiration: https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Organizing_Hyperparameter_Sweeps_in_PyTorch_with_W%26B.ipynb#scrollTo=r4VjKui20N3j
+"""
+
+from utils import read_json
+import wandb
+
+sweeps = read_json("sweep_config.json")
+
+
+def train(config=None):
```

Comment:
I think it is better to use the HF trainer instead. Here we can e.g. use 16 bit precision (as opposed the 32 or 64) or use the deepspeed library. I assume there is also other optimisations.